doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,730 @@
 Scientific Computing and Visualization with Spyder
 ##################################################
 
-This is a work in progress. The workshop for Scientific Computing and Visualization with Spyder will be available soon!
+This workshop is a tour of the features that make Spyder an ideal IDE for using the scientific tools offered by Python. Throughout the workshop, we will apply the scientific method to answer some questions related to our preferences and cognitive abilities.
+
+
+
+=============
+Prerequisites
+=============
+
+To follow this workshop we recommend that you have a basic knowledge of Python. You can visit `The Python Tutorial`_ to learn the basics of this programming language or to refresh your knowledge of Python.
```

Comment:
I'm not quite sure but I think it should be "some basic knowledge" but perhaps Melissa can correct me if I'm wrong. 
Also I would remove "or to refresh your knowledge of Python." to avoid repetition.